apps/minaintyg/src/components/PageHeading/PageHeading.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { PageHeading } from './PageHeading'
+
+it('should remove clamped when show more button is pressed', async () => {
+  render(<PageHeading heading="Test">Lorem ipsum</PageHeading>)
```

Comment:
More tests can be added to see that "Visa mer", "Visa mer text", children, heading etc. are rendered